names/data/names_data.hcl

```diff
@@ -0,0 +1,16131 @@
+# Copyright (c) HashiCorp, Inc.
+
+service "accessanalyzer" {
+
+  sdk {
+    id             = "AccessAnalyzer"
+    client_version = [2]
+  }
+
+  names {
+    aliases             = [""]
+    provider_name_upper = "AccessAnalyzer"
+    human_friendly      = "IAM Access Analyzer"
+  }
+
+  client {
+    go_v1_client_typename = "AccessAnalyzer"
+    skip_client_generate  = false
```

Comment:
Maybe reverse this parameter (e.g. `generate_client`) and default to `true`, so it only needs to be set when not generating the client

Reply:
Attributes that have false values are omitted. How do you feel about this?

Reply:
Not a blocker, but typically we should avoid double-negative parameters, since the logic is inverted. In this case, the `SkipClientGenerate` parameter was added after the fact and was true for only a small set of services, so it was used.